Source/Blazorise/BlazoriseOptions.cs

```diff
@@ -124,5 +124,23 @@ public string LicenseKey
     /// </summary>
     public bool SafeJsInvoke { get; set; } = true;
 
+    /// <summary>
+    /// Gets or sets the options used to configure the behavior of the Tooltip. This allows you to customize the Tooltip's appearance and timing.
+    /// </summary>
+    public BlazoriseTooltipOptions TooltipOptions { get; set; }
+
     #endregion
+}
+
+/// <summary>
+/// Represents the options available for configuring the behavior of a Tooltip in Blazorise.
+/// </summary>
+public class BlazoriseTooltipOptions
```

Comment:
I was more of a separate file for every class in the past and I believe I've seen similar reviews from you in the past.
Personally, nowadays I don't mind keeping related stuff in the same file. What about you? Are you fine with keeping both of these in the same file?

Reply:
Done